addons/wardrobe/functions/fnc_compare_components.sqf

```diff
@@ -0,0 +1,49 @@
+#include "../script_component.hpp"
+
+/*
+ * Author: OverlordZorn
+ * Function to compare the components of the current and target wearables.
+ *
+ * Arguments:
+ * 0: current variant <CONFIG>
+ * 1: desired variant <CONFIG>
+ *
+ * Return Value:
+ * Nested array <ARRAY>
+ * - configs of missing components <ARRAY>
+ * - configs of surplus components <ARRAY>
+ *
+ * Example:
+ * [_cfg_origin, _cfg_tgt] call ace_wardrobe_fnc_compare_components
+ *
+ * Public: No
+ */
+
+params ["_cfg_origin", "_cfg_tgt"];
```

Comment:
Lots of snake_case around the SQF bits, change to camelCase where possible. GVARs included.